odoo/addons/base/models/ir_translation.py

```diff
@@ -109,9 +109,11 @@ def finish(self):
                            WHERE type = 'code'
                            AND noupdate IS NOT TRUE
                            ON CONFLICT (type, lang, md5(src)) WHERE type = 'code'
-                            DO UPDATE SET (name, lang, res_id, src, type, value, module, state, comments) = (EXCLUDED.name, EXCLUDED.lang, EXCLUDED.res_id, EXCLUDED.src, EXCLUDED.type, EXCLUDED.value, EXCLUDED.module, EXCLUDED.state, EXCLUDED.comments)
-                            WHERE EXCLUDED.value IS NOT NULL AND EXCLUDED.value != '';
-                       """ % (self._model_table, self._table))
+                            DO UPDATE SET (name, lang, res_id, src, type, value, module, state, comments) = (EXCLUDED.name, EXCLUDED.lang, EXCLUDED.res_id, EXCLUDED.src, EXCLUDED.type, EXCLUDED.value, EXCLUDED.module, EXCLUDED.state,
+                                                                                                             CASE WHEN %s.comments = 'openerp-web' THEN 'openerp-web' ELSE EXCLUDED.comments END
+                                                                                                            )
+                            WHERE EXCLUDED.value IS NOT NULL AND EXCLUDED.value != ''
+                       """ % (self._model_table, self._table, self._model_table))
```

Comment:
@odony in your [patch](https://github.com/odoo/odoo/pull/60810#discussion_r515311762) it was

                       """ % (self._model_table, self._table, self._table))

But having ``self._table`` at the end of the list makes another statement equal to 

      CASE WHEN EXCLUDED.comments = 'openerp-web' THEN 'openerp-web' ELSE EXCLUDED.comments END

Am I correct? Should it be ``self._model_table`` as I did in this PR?

Reply:
ping @odony